python/cuml/cuml/linear_model/ridge.pyx

```diff
@@ -31,7 +30,6 @@ from cuml.common import input_to_cuml_array
 from cuml.common.array_descriptor import CumlArrayDescriptor
 from cuml.common.doc_utils import generate_docstring
 from cuml.internals.api_decorators import (
-    api_base_return_array_skipall,
```

Comment:
were those just unused prior?

Reply:
Yeah, not sure why the linter only hit these now rather than before. There were a couple things like this throughout.